WordPress/Classes/ViewRelated/Me/Me Main/MeViewController.swift

```diff
@@ -178,7 +178,7 @@ class MeViewController: UITableViewController {
                                               loading: sharePresenter.isLoading))
 
                 rows.append(NavigationItemRow(title: RowTitles.about,
-                                              icon: UIImage.gridicon(.mySites),
+                                              icon: UIImage.gridicon(AppConfiguration.isJetpack ? .plans : .mySites),
```

Comment:
If you have a better idea here (than using an icon called `.plans`), please let me know. This icon comes from our icon set, https://github.com/Automattic/gridicons. One option could be duplicating this icon and renaming the copy `jetpack`, but Gridicons appears to be deprecated (see p1666277353002919-slack-C3EA6SMFH and threads below it), so updating the icon set didn't seem like the best option.

Reply:
I agree that we need a wider discussion of the alternatives if `gridicons` are duplicated. For now, the solution works. If we do replace `gridicons` with something, this place will get replaced as well.

Reply:
There is some discussion about improving (in the future) this icon set situation: pcdRpT-139-p2#comment-1521